spec/index.html

```diff
@@ -461,7 +461,7 @@ <h2>RDF Documents and Syntaxes</h2>
       many different ways to encode the same <a>RDF graph</a> or
       <a>RDF dataset</a>, for example through the use of
       <a>namespace prefixes</a>,
-      <a>relative IRI references</a>, <a>blank node identifiers</a>,
+      <a>IRI references</a>, <a>blank node identifiers</a>,
       and different ordering of statements. While these aspects can have great
```

Comment:
Unrelated: should that be "triples"?

Reply:
@afs -- Which "that"?

Reply:
"statements"

Reply:
I wouldn't argue with changing `ordering of statements` but given that this is talking about `ways to encode the same <a>RDF graph</a> or <a>RDF dataset</a>`, I think it must become `ordering of triples and/or quads` or possibly `ordering of triples (and/or quads)`.

I see no strong reason not to make that tiny additional change within this PR, but if anyone objects to that, I ask that you create an issue referencing this discussion.

Reply:
Changed to "triples". "quads" aren't appropriate in this context, and N-Quads actually uses "statements" in some circumstances where N-Triples uses "triples". In this context, "triples" is correct.

Reply:
That would be tricky. Quads only appear in non-normative 4.3.

The definition of RDF Datasets is about graphs, and those graphs are sets of triples.